db/migrate/20250124154917_create_live_result_tables.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateLiveResultTables < ActiveRecord::Migration[7.2]
+  def change
+    create_table :live_results do |t|
+      t.references :registration, null: false, index: true
+      t.references :round, null: false, index: true
+      t.references :entered_by, type: :integer, null: false, foreign_key: { to_table: :users }
+      t.datetime :entered_at, null: false
```

Comment:
Is it sufficient to keep this information per scorecard?
I'm thinking about FMC and MBLD where you enter results with long gaps in between, and these gaps also make it plausible that individual scores are entered by different people.

Reply:
Also related about the timestamp: I assume this is the time when the scorecard was first inserted, but what if we just change one score later? Where is that being recorded?

Reply:
I was thinking I would just overwrite entered_by and entered_at with the latest person. I think that's how jonatan does it